keanu-project/src/test/java/io/improbable/keanu/e2e/regression/RadonHierarchicalRegression.java

```diff
@@ -48,28 +54,13 @@ public void canPerformSimpleLinearRegression() {
     }
 
     @Test
-    public void canPerformRegressionWithOneHierarchy() {
```

Comment:
A hierarchy with one grouping is basically a simple linear regression, it should be able to find this. Does this test fail?

Reply:
have re-added and this passes